cookbooks/aws-parallelcluster-slurm/recipes/upgrade_slurm.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
If we want to put the script in the code, we can consider move it to another folder